R/bayesplot-colors.R

```diff
@@ -310,34 +357,8 @@ is_hex_color <- function(x) {
   isTRUE(nchar(x) == 7)
```

Comment:
ggplot2 will take nine character ones where the last two set the alpha level. Some palettes emit colors with these extra characters.

```
scales::viridis_pal()(4)
#> [1] "#440154FF" "#31688EFF" "#35B779FF" "#FDE725FF"
```

Not sure we need to accommodate this though.

Reply:
Oh, I didn't know that. I'm not sure we need to accommodate it either, although I'm not opposed. I suppose adding support for that can be a separate issue if we every want to do it or get a request.